postgres-dataprovider-plugin/README.md

```diff
@@ -0,0 +1,85 @@
+## Configuration docs for Postgres Plugin
+
+- Pre-requisites
+  Authorisation Provider should expose the unique identifier in the `sub` field of the JWT token.
+  Eg. If one is using eSignet with mock-identity-system:0.10.0 and above it can be achieved by setting:
+  ```mosip.mock.ida.kyc.psut.field=individualId```
+  where individualId will be the identifier to locate the identity in the expected identity registry.
+
+1. Create the tables with all the fields that should be returned by the Postgres Data Provider Plugin within the certify postgres database.
+    - Refer the following query for insertion in DB:
+      ```
+        CREATE TABLE certify.<table_name> (
+                 attribute_1 <type> NOT NULL,
+                 attribute_2 <type> NOT NULL,
+                 ...
+                 CONSTRAINT pk_id_code PRIMARY KEY (<identifier>)
+             );
+      ```
+
+2. The schema context containing all the required fields should be hosted in a public url.
+    - Refer this link for an existing context: [Registration Receipt Schema](https://piyush7034.github.io/my-files/registration_receipt.json)
+      Eg: https://<username>.github.io/<project_name>/<file_name>.json
+    - Also change the respective credential name:
+      ```
+         {
+             "@context": {
+                 "@version": 1.1,
+                 "@protected": true,
+                 "type": "@type",
+                 "schema": "https://schema.org/",
+                 "<credential_name>": {
+                     "@id": "https://<username>.github.io/<project_name>/<file_name>.json#<credential_name>"
+                 },
+                 <field1>: "schema:<type>"
+                 <field2>: "schema:<type>"
+                 ...
+             }
+         }
+      ```
+    - The primary_key should be a identifier that is existing in the current mock_identity_system records.
```

Comment:
lets not stress too much on the mock_identity_system, since it is only one reference impl and the actual places this plugin to be used might not sue the mock identity system

Reply:
Removed reference of mock-identity-system as it may not be a dependency every time.